cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,77 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticlePage from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
+import '../support/commands';
 
-describe('Article', () => {
-  before(() => {
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
+
+describe('Article Page', () => {
+  let user;
+  let title;
+  let description;
+  let content;
+  let tags;
 
+  before(() => {
```

Comment:
The database is being cleared only once before all tests. It would be better to clear the database before each test to avoid data collision and to make sure each test starts with a clean state. You can do this by moving `cy.task('db:clear');` into 'beforeEach' block.

Reply:
I made it because Marina Bidenko wrote:
It's not recommended to clear the database before each test. It might cause performance issues. Consider resetting the state in the database only when necessary.
[https://github.com/mate-academy/qa_cypress_e2e_project/pull/59#discussion_r1285019476](url)